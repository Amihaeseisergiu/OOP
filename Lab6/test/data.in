  10  ,23  ,  1 , 4  , 5,  60,1,10
