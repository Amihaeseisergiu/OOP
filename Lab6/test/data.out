1
4
5
10
23
60
